build_tools/cmake/iree_lit_test.cmake

```diff
@@ -52,10 +50,17 @@ function(iree_lit_test)
   get_filename_component(_TEST_FILE_PATH ${_RULE_TEST_FILE} ABSOLUTE)
 
   list(TRANSFORM _RULE_DATA REPLACE "^::" "${_PACKAGE_NS}::")
+  list(TRANSFORM _RULE_TOOLS REPLACE "^::" "${_PACKAGE_NS}::")
   set(_DATA_DEP_PATHS)
-  foreach(_DATA_DEP ${_RULE_DATA})
+  foreach(_DATA_DEP IN LISTS _RULE_DATA _RULE_TOOLS)
     list(APPEND _DATA_DEP_PATHS $<TARGET_FILE:${_DATA_DEP}>)
-  endforeach(_DATA_DEP)
+  endforeach()
+
+  set(_LIT_PATH_ARGS)
+  # TODO: only tools
+  foreach(_TOOL IN LISTS _RULE_TOOLS)
+    list(APPEND _LIT_PATH_ARGS "--path" "$<TARGET_FILE_DIR:${_TOOL}>")
+  endforeach()
```

Comment:
TODO still relevant? Add a name and a bit more context?

Reply:
Thanks for catching. I've gotten in the habit of sticking bare TODOs in to remind me to fix something before submission/sending for review, but we don't have the presubmit here that yells at you for bare TODOs :-D